tests/support/util.tcl

```diff
@@ -485,8 +485,9 @@ proc find_available_port {start count} {
             set port $start
         }
         set fd1 -1
-        if {[catch {set fd1 [socket -server 127.0.0.1 $port]}] ||
-            [catch {set fd2 [socket -server 127.0.0.1 [expr $port+10000]]}]} {
+        proc dummy_accept {chan addr port} {}
+        if {[catch {set fd1 [socket -server dummy_accept -myaddr 127.0.0.1 $port]}] ||
+            [catch {set fd2 [socket -server dummy_accept -myaddr 127.0.0.1 [expr $port+10000]]}]} {
```

Comment:
i just tested, disabling the other fixes (close in introspection.tcl, and re-adding the include in the blockedclient.tcl), when i also remove the `127.0.0.1` in util, the test hangs, and when i keep it, the test passes.
so i think the current version is what we should merge, unless you think it can cause problems.

Reply:
no, i don't know what happened to macos, which has strange behavious.
i dont find a good way to solve it

```
# start server
$ `./src/redis-server`

# get config from another console
# please note that we can connect Redis server success here.
$ redis-cli config get bind
1) "bind"
2) "* -::*"

# start a fake server in tclsh, and success.
% socket -server dummy_accept -myaddr 127.0.0.1 6379

# it will hang here, because cli connects to the fake server
$ redis-cli 
```

Reply:
@oranagra i think we can skip this strange issue with macos that has nothing to do with redis itself.

Reply:
thta's odd, redis started first, and yet redis-cli connects to the fake server that (successfully) started later?

anyway, our tests explicitly bind to 127.0.0.1, so maybe it doesn't matter to us?

Reply:
yes, and the cli success connectes to redis-server at first, after the fake server starts, the cli connects to the fake server, and I don't stop redis-server.
let's skip it now.